pages/api/vipps/addAll.ts

```diff
@@ -46,24 +46,27 @@ export default async function handler(req, res) {
       }>[] = [{ deleteMany: { filter: {} } }];
 
       for (const row of rows) {
-        const [dateString, brutto, firstName, lastName] = row.split(",");
+        const values = row.split(",");
+
+        const amount = values[6];
+        const firstName = values[14];
+        const lastName = values[14];
```

Comment:
I think you meant this
```suggestion
        const lastName = values[15];
```

Reply:
yessir :+1: